.github/workflows/cd_gestor_rutas_entrega.yml

```
@@ -0,0 +1,98 @@
+  push:
+    branches:
+      - main
+    paths:
+      - 'backend/logistica_servicio/generador_rutas_entrega/**'
+      - '.github/workflows/cd_gestor_rutas_entrega.yml'
+  workflow_dispatch:
+jobs:
+   deploy_to_gke:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)